app/views/resources/_resource.html.erb

```diff
@@ -1,6 +1,13 @@
 <tr>
+  <td class="text-end">
+    <% if resource[:thumbnail].present? %>
+      <%= image_tag resource[:thumbnail], alt:'', class: 'resource-thumbnail', skip_pipeline: true %>
```

Comment:
Asking from a place of ignorance: is it better to provide an empty `alt` attr or to provide a relatively meaningless one like `'Thumbnail for resource'`?

Reply:
I think the thumbnail is pretty arguably decorative, not informative, and the w3c recommends an empty alt tag:
  https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
@cbeer OK, sold.

Reply:
@mjgiarlo It depends on context but in general if you can't provide a meaningful `alt` value, you should leave it empty. An empty `alt` won't be read by a screenreader but one with a value with be. So hearing `Thumbnail for resource` probably doesn't add value for the screenreader user (especially hearing it repeated in close succession when reading through a list of resources) so it's better to leave empty.

Also, in this context, an argument could be made that even if we could provide unique descriptive values for the `alt` attribute of each resource, that value is probably not helping the screenreader user to distinguish one resource in the list from another very much (since the title is read immediately after and is likely to be more helpful in distinguishing) so that's another reason for leaving the `alt` attribute empty in this context.

Reply:
@ggeisler Much obliged!